iocore/hostdb/HostDB.cc

```diff
@@ -1037,6 +1037,33 @@ HostDBProcessor::getbyname_imm(Continuation *cont, process_hostdb_info_pfn proce
   return &c->action;
 }
 
+Action *
+HostDBProcessor::getall(Continuation *cont)
```

Comment:
Bad name. This should be "iterate" because that's what it is doing or "probeAll" to be consistent with the event / method name.